app/assets/stylesheets/header.scss

```diff
@@ -1,8 +1,16 @@
 .header{
-  border-bottom:solid 2px #5DC9C9;
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  border-bottom:  2px solid #dcdcdc;
+  border-radius:  0 0 5px 5px;
+  box-shadow: 0 1px 1px #cecdcd inset;
```

Comment:
Color literals like `#cecdcd` should only be used in variable declarations; they should be referred to via variable everywhere else.